src/WaveClip.cpp

```diff
@@ -100,19 +100,8 @@ WaveClip::WaveClip(const WaveClip& orig,
    mEnvelope = std::make_unique<Envelope>(*orig.mEnvelope);
 
    if ( copyCutlines )
-      // Copy cutline clips that fall in the range
-      for (const auto &ppClip : orig.mCutLines)
-      {
-         const WaveClip* clip = ppClip.get();
-         double cutlinePosition = orig.GetSequenceStartTime() + clip->GetSequenceStartTime();
-         if (cutlinePosition >= t0 && cutlinePosition <= t1)
-         {
-            auto newCutLine =
-               std::make_unique< WaveClip >( *clip, factory, true );
-            newCutLine->SetSequenceStartTime( cutlinePosition - t0 );
-            mCutLines.push_back(std::move(newCutLine));
-         }
-      }
+      for (const auto &cutline : orig.mCutLines)
+         mCutLines.push_back(std::make_unique<WaveClip>(*cutline, factory, true));
```

Comment:
So is it a consequence of this change:

I can make a clip; make a cutline; drag the smart clip boundary to hide the cutline; copy a part of the clip; paste to a new track; drag the boundary of the new clip -- and find the duplicated cutline too?

Reply:
For some reason cutlines aren't copied with regular copy-paste, but they should be copied when part of a clip that contains a cutline is duplicated. But in general what you said is correct.